src/Parser/MarksmanshipHunter/Modules/Features/CooldownTracker.js

```diff
@@ -12,6 +14,19 @@ class CooldownTracker extends CoreCooldownTracker {
       ],
     },
   ];
+
+  castEventSpells = [
+    SPELLS.WINDBURST_MOVEMENT_SPEED.id,
+  ];
+
+  on_byPlayer_cast(event) {
+    const spellID = event.ability.guid;
+    if (this.castEventSpells.indexOf(spellID) !== -1)  {
```

Comment:
This could be `this.castEventSpells.includes(spellID)` which is a little more readable and works the same.